packages/snake/helloworld/apps.py

```diff
@@ -0,0 +1,88 @@
+import gspread
+import csv
+import firebase_admin
+from firebase_admin import credentials, firestore
+
+cred = credentials.Certificate("clauth.json")
+firebase_admin.initialize_app(cred)
+gc = gspread.service_account("auth.json")
+db = firestore.client()
+
+members = gc.open("All Quarters")
+leads = gc.open("All PLead Apps")
+
+member_list = members.worksheets()
+lead_list = leads.worksheets()
+
+member_title = ["application_id", "applicant_name", "applicant_email", "applicant_year",
+                "applicant_major", "status", "quarter", "timestamp", "first_choice",
+                "second_choice", "third_choice"]
+lead_title = ["project_id", "project_name", "project_description", "status", "lead_name",
+            "lead_email", "lead_year", "open_spots"]
+
+def store_data(list, update):
+    id = 0
```

Comment:
why is the id of the object a number? how does this work on the second run? are you going to overwrite data?